src/components/Settings/AllSettingsFields.tsx

```diff
@@ -276,25 +276,6 @@ export const AllSettingsFields = forwardRef(
               , and start a discussion if you don't see it! Your feedback will
               help us prioritize what to build next.
```

Comment:
@franknoirot Unrelated: are we still good with the copy there and the link to https://github.com/KittyCAD/modeling-app/discussions?

Maybe cc @jessfraz too. Just curious how we want to approach voting for features etc. moving forward